knimin/lib/data_access.py

```diff
@@ -4218,6 +4222,24 @@ def read_shotgun_plate(self, plate_id):
             res = dict(res[0])
             res['condensed_plates'] = [eval(v)
                                        for v in eval(res['condensed_plates'])]
+            # Get positions
+            sql = """SELECT row, col, sample_id, dna_concentration
+                     FROM pm.shotgun_plate_layout
+                     WHERE shotgun_plate_id = %s"""
+            TRN.add(sql, [plate_id])
+            values = TRN.execute_fetchindex()
+            # getting rows/cols and discarding them from res
+            rows = res['rows']
+            del res['rows']
```

Comment:
cause I'm actually using them below to create the data structure, I could put it in another SQL but thought this will be faster ...